src/index.ts

```
@@ -265,6 +266,11 @@ export async function signAddon({
       );
     }
   }
+  if (!output && channel === 'listed') {
```

Same issue here, can we throw an error here? or at least set the `exitCode` to non-zero so that the next task will know that we don't get a file downloaded.

---

Thing is, in my CI I'm submitting an add-on that needs manual review. I don't think it's right for the CI to fail if manual review is needed.

---

> Same issue here, can we throw an error here? or at least set the `exitCode` to non-zero so that the next task will know that we don't get a file downloaded.

Do you think we could add a flag to toggle the behavior of silently not downloading a file? What should it be named in the function and in the CLI?

---

I see, that makes sense. Now I got why you proposed to return immediately when `output` is empty.

---

> Do you think we could add a flag to toggle the behavior of silently not downloading a file?

Yes, that would be a good choice. But since we know what we passed, it's also acceptable to handle the output differently by ourselves as the users.